user-guide/Troubleshooting/Known_issues/KI_SNMPv3_elements_broken_after_upgrade.md

```diff
@@ -14,7 +14,9 @@ When upgrading from a 10.4.0 Main Release version starting from 10.4.0 [CU6] to
 
 ## Fix
 
-No fix is available yet.<!-- RN 41630 -->
+This issue will no longer occur from DataMiner 10.5.1 onwards.<!-- RN 41630 -->
+
+However, note that once you have encountered the issue, you will need to apply the [workaround](#workaround) below to resolve this. Simply upgrading to 10.5.1 is not sufficient.
```

Comment:
If you have elements that broke during the upgrade to e.g. 10.4.9, then another upgrade to 10.5.0 or 10.5.1 will fix the issue. So upgrading to 10.5 CU0 or 10.5.1 is sufficient to resolve the problem.

Reply:
In that case, I think we'd better not mention the workaround in the [release note entry](https://docs.dataminer.services/release-notes/General/General_Feature_Release_10.5/General_Feature_Release_10.5.1.html#problem-with-snmpv3-elements-after-upgrading-from-main-release-version-1040-cu6-or-later-to-feature-release-version-104x-1049-to-10412-id-41630), because this will give people the impression that they do need to apply it after all. @spetebroodt, FYI.

Reply:
Workaround removed and RN added in both main release and feature release.